contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -124,15 +124,18 @@ In this case, only the `root` component can be accessed in Vue while the rest ar
 
     This follows the [1-way data flow](https://vuejs.org/v2/guide/components-props.html#One-Way-Data-Flow) encouraged by Vue, which
     ensures that data can only be changed by the component itself and also allows bugs to be easily traced in the code.
+    
+    To pass props to a Vue component, `v-bind:<prop-name>` is used. A demonstration of passing props is shown in the code segment below:
```

Comment:
Suggestion:

Find another example to show `v-bind` since `v-for` is not introduced anywhere. Alternatively, add a section talking about conditional rendering with `v-if` or `v-for`.